habitat-baselines/habitat_baselines/rl/hrl/skills/nn_skill.py

```diff
@@ -128,6 +128,7 @@ def _internal_act(
         masks,
         cur_batch_idx,
         deterministic=False,
+        new_action=None
```

Comment:
You added new_action to _internal_act() function for all skill classes. Why do we want this?

Reply:
The new_action is for saving the 3d pos of the new action, I need to add the params into high level skill class to pass this param.